spotbugs-exclude.xml

```diff
@@ -3,4 +3,34 @@
     <Match>
         <Package name="~io\.dapr\.examples.*"/>
     </Match>
+
+    <!--Temporarily ignoring checking after upgrade to new spotbugs version-->
+    <Match>
+        <Package name="~io\.dapr.*"/>
+        <Bug pattern="EI_EXPOSE_REP"/>
+    </Match>
+
+    <!--Temporarily ignoring checking after upgrade to new spotbugs version-->
+    <Match>
+        <Package name="~io\.dapr.*"/>
+        <Bug pattern="CT_CONSTRUCTOR_THROW"/>
+    </Match>
+
+    <!--Temporarily ignoring checking after upgrade to new spotbugs version-->
+    <Match>
+        <Package name="~io\.dapr.*"/>
+        <Bug pattern="SS_SHOULD_BE_STATIC"/>
+    </Match>
+
+    <!--Temporarily ignoring checking after upgrade to new spotbugs version-->
+    <Match>
+        <Package name="~io\.dapr.*"/>
+        <Bug pattern="EI_EXPOSE_REP"/>
+    </Match>
+
+    <!--Temporarily ignoring checking after upgrade to new spotbugs version-->
+    <Match>
+        <Package name="~io\.dapr.*"/>
+        <Bug pattern="EI_EXPOSE_REP2"/>
+    </Match>
```

Comment:
is there a reason why we need this?

Reply:
https://github.com/dapr/java-sdk/issues/971

See “Spotbugs bug pattern” section in Appendices  part.